Flask_webapp/templates/index.html

```diff
@@ -46,25 +24,25 @@
 <!--  -->
 <!--  -->
 <body>
-	<div>
-		<br><br><br><br>
-		<form>
-	    <label for="tickerInput">Ticker: </label>
-	    <input type="text" id="tickerInput" placeholder="Stock Ticker">
-		  <button type="submit">Submit</button>
-		</form>
-		<br>
-		<div id="successAlert" role="alert" style="display:none;"></div>
-		<div id="errorAlert" role="alert" style="display:none;"></div>
-	</div>
-
-	<br><br>
-	<div>STILL DOESN"t catch the wrong ticker ERROR!!</div>
+<div>
+	<br><br><br><br>
+	<form class="form-class">
+    <label for="tickerInput">Ticker: </label>
+    <input type="text" id="tickerInput" placeholder="Stock Ticker">
+	  <button type="submit">Submit</button>
+	</form>
+	<br>
+	<div id="successAlert" role="alert" style="display:none;"></div>
+	<div class="errorMsg-class" id="errorAlert" role="alert" style="display:none;"></div>
+</div>
+<hr class="hr-class" />
+<br><br>
+<div>STILL DOESN"t catch the wrong ticker ERROR!!</div>
 
 <!-- New division -->
 <br>
 <div>
-<input type="button" onclick="location.href='/price_predict';" value="Predict Stock Price" />
+<input type="button" onclick="location.href='/price_predict';" value="Go to Predict Stock Price" />
```

Comment:
instead of on onclick method you can just wrap this in an `<a>` tag to make it a link
```
<a href="/price_predict"><button>Go to predict stock prive</button></a>
```